src/loaders/express.loader.ts

```diff
@@ -23,43 +23,72 @@ export class ExpressLoader extends AbstractLoader {
     );
     loadPackage('express-formidable', '@admin-bro/nestjs');
 
+    let router;
+
+    if ('auth' in options) {
+      loadPackage('express-session', '@admin-bro/nestjs');
+      router = adminBroExpressjs.buildAuthenticatedRouter(
+        admin, options.auth, undefined, options.sessionOptions, options.formidableOptions
+      );
+    } else {
+      router = adminBroExpressjs.buildRouter(admin, undefined, options.formidableOptions);
+    }
+
+    app.use(options.adminBroOptions.rootPath, router);
+    this.reorderRoutes(app, options);980
```

Comment:
```suggestion
    this.reorderRoutes(app, options);
```

Reply:
God damn it :laughing: My cat wanted to add something too :cat: